tests/bluetooth/audio_shell/prj.conf

```diff
@@ -17,9 +17,8 @@ CONFIG_BT_MAX_CONN=4
 
 CONFIG_BT_AUDIO=y
 CONFIG_BT_AUDIO_UNICAST=y
-CONFIG_BT_AUDIO_BROADCAST=n
-CONFIG_BT_BAP=n
-CONFIG_BT_ISO_BROADCAST=n
+CONFIG_BT_AUDIO_BROADCAST=y
+CONFIG_BT_BAP=y
```

Comment:
Also re-enable BASS and BASS_CLIENT (with debugging)?

Reply:
Fixed